lib/src/monorepo_command_runner.dart

```diff
@@ -25,8 +23,23 @@ class MonorepoCommandRunner extends CommandRunner<Null> {
           "moronepo",
           "A tool to simplify development in a dart multi-package repository.",
         ) {
-    argParser.addOption("working-directory", abbr: "d", help: "specifies the working directory");
+    argParser.addOption("working-directory", abbr: "w", help: "specifies the working directory");
```

Comment:
What do you think about use full word, not a shortcut abbr?

Reply:
You can use both this way - `--working-directory` and `-w` work fine.